docs/userGuide/syntax/horizontalrules.mbdf

```diff
@@ -11,6 +11,57 @@ ______________
 </variable>
 </include>
 
+### Different Types Of Horizontal Rules
+
+Add additional attributes and classes to modify the style, thickness, and color of a horizontal line. 
+
+Available style attributes:
+- dotted
+- dashed
+- double
+
+Available size attributes:
+- thick
+- thick-1
+- thick-2
+- thick-3
+
+Available color classes (use any of the available BootStrap border color classes):
+- border-info
+- border-primary
+- etc
+
+
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
```

Comment:
Should we split this into the above sections? 10 lines at once might be a bit much to keep track of

Reply:
Sure, I have split them into individual code blocks under each description (pls see the deployment preview [here](https://deploy-preview-1685--markbind-master.netlify.app/userguide/formattingcontents#horizontal-rules))